libraries/serf_helper.rb

```diff
@@ -1,22 +1,15 @@
 # coding: UTF-8
 require 'json'
 
-class Chef::Recipe::SerfHelper < Chef::Recipe
+class Chef::Recipe::SerfHelper
 
   SERF_VERSION_REGEX = /^Serf v\d.\d.\d/
   VERSION_REGEX = /\d.\d.\d/
 
-  # Initializes the helper class
-  def initialize chef_recipe
-    super(chef_recipe.cookbook_name, chef_recipe.recipe_name, chef_recipe.run_context)
+  attr_accessor :node
 
-    # TODO: Support other distributions besides 'linux'
-    node.default["serf"]["binary_url"] = File.join node["serf"]["base_binary_url"], "#{node["serf"]["version"]}", "serf_#{node["serf"]["version"]}_linux_#{node["serf"]["arch"]}.zip"
```

Comment:
I still like having this attribute as it gives you an override if you need it. This was how I used to handle attributes made of other attributes before Chef 11/12. Mind adding this back in and moving it into the attributes file and switching it back to use the attribute?